generator/internal/parser/openapi_test.go

```diff
@@ -606,7 +593,7 @@ func TestOpenAPI_MakeAPI(t *testing.T) {
 				Name:          "locations",
 				JSONName:      "locations",
 				Documentation: "A list of locations that matches the specified filter in the request.",
-				Typez:         api.MESSAGE_TYPE,
+				Typez:         11,
```

Comment:
Why replace this with the literal instead of the constant?

Reply:
This is a typo. Replaced throughout the file.